doc/source/conf.py

```diff
@@ -41,6 +42,15 @@
     '.md': 'recommonmark.parser.CommonMarkParser',
 }
 
+# Generate JSON version of the helm-chart
+import yaml, json, sys
+import requests
+resp = requests.get('https://raw.githubusercontent.com/jupyterhub/helm-chart/master/jupyterhub/schema.yaml')
```

Comment:
This feels quite bad to me, since this won't get rebuilt every time that changes, there's no tagging, etc.

I think we should either:

1. Put this docs in the helm-chart repo
2. Merge the helm-chart repo into this repo

Before we do this. Quickest thing to do is (1), but I'm also happy to just do (2). I'd rather merge helm-chart repo into this, since the branding as 'z2jh' seems to have stuck a lot more than just 'helm chart'

Reply:
yeah I agree this is hacky...let's do number 2 since it seems like this is the direction in which we're headed anyway. What if we renamed the repository to `jupyterhub-k8s` and then `zero to jupyterhub k8s` would be the documentation for it? What do you think @willingc or @minrk ?

Reply:
I'm fine with merging in the helm chart. Is there a technical reason to rename the repo? If not, let's leave it as is.

Reply:
Let's merge the helm-chart here and leave the name as is!

Reply:
@yuvipanda Any downsides that you can see down the road from merging the helm chart here? I'm not thinking of any but 🤷‍♀️

Reply:
Licensing is the only one. The helm-chart repo is Apache2 explicitly because that's the upstream charts repo license, and that's where we want to eventually be. This repo is 3BSD. We can dual license this as apache2 and 3BSD to get around that.

Reply:
If @minrk is cool with the dual license, then let's do it. Thanks.

Reply:
the only reason I was thinking of renaming is that in my mind the helm chart is the tech product, and "zero to jupyterhub k8s" is documentation around that product. So in my mind calling the repo `jupyterhub-k8s` would designate this space as 'jupyterhub development around kubernetes'...and that this could be more than the zero to jupyterhub guide. This is just my own idiosyncratic way of thinking about this...I'm happy to keep the naming as-is

Reply:
Most of the work of actually integrating with kubernetes (for some definition of 'actually') is in kubespawner repo, so I think we're ok!

Reply:
sounds good to me :-)  lemme know if/when you merge in helm-chart and then I can rebase / update this PR

Reply:
@choldgraf Let's try to keep things simple by not renaming. There's enough churn and changes in other places. Let's not introduce more than necessary.

Reply:
how about we rename the repo to "zero to zero to jupyterhub k8s for kubernetes"? :-)

Reply:
(jk, current name sounds good to me)